### PR TITLE
k8smeetup-azure&kubespray-pr-2017-8-15

### PR DESCRIPTION
**Pull request by yufengqi**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5365)

### CODE REVIEW

**Comment by yufengqi** [comment]
Registered

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/5365#-:-Kt_onCi6AaPtoq-R84S:b-li6x09)*

File: cn/docs/getting-started-guides/kubespray.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+* 目标服务器通过 SSH 连接（tcp / 22）直接连接到您的节点或通过 bastion host/ssh 跳转框
+
+
+
+* 目标服务器需要有特权用户
+
+
+
+* 您的 SSH 密钥必须复制到您 inventory 上的所有服务器
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/getting-started-guides/kubespray.md]: 服务器->节点 or 主机
that seems to be better.

File: cn/docs/getting-started-guides/kubespray.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+可以使用 [reset 命令](https://github.com/kubernetes-incubator/kubespray/blob/master/reset.yml)重置节点和清除使用 Kubespray 安装的所有组件。
+
+**注意:** 使用 reset 命令时，确保不是生产集群！
+{. :caution}
+
+
+
+## 反馈
+
+* Slack 通道: [kubespray](https://kubernetes.slack.com/messages/kubespray/)
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/getting-started-guides/kubespray.md]: 通道
->
频道。
slack是一款在线通讯工具。

File: cn/docs/getting-started-guides/azure.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+更多内容请参考快速入门 **[ACS-Engine Kubernetes 教程](https://github.com/Azure/acs-engine/blob/master/docs/kubernetes.md)**。
+
+
+
+##  在 Azure 上使用 CoreOS Tectonic
+
+
+
+基于 Azure 的 CoreOS Tectonic Installer 是**开源**的，可以在 GitHub 上使用和贡献：**[Tectonic Installer](https://github.com/coreos/tectonic-installer)**。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/getting-started-guides/azure.md]: 可以在 GitHub 上使用和贡献
->
可以通过GitHub 获取并使用，也可以通过Github进行社区贡献

File: cn/docs/getting-started-guides/kubespray.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+
+
+Kubespray 允许自定义部署以下方面:
+
+
+
+* CNI (网络)插件
+* DNS 配置
+* 控制面板的选择：本机/二进制或通过 docker 或 rkt 集装箱
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/getting-started-guides/kubespray.md]: 集装箱？
这个错误太明显了。

**Comment by Bradamant3** [comment]
@yufengqi it looks as though your CNCF CLA isn't properly processed. Can you please check? The correct label is automatically applied and the check passes when it's done.

**Comment by zhangxiaoyu-zidif** [comment]
@yufengqi 
Hi, please sign CLA, thanks.

**Comment by steveperry-53** [comment]
ping @yufengqi

**Comment by steveperry-53** [comment]
I suggest we keep this open until Friday, 10/6/17. Then if we still don't have a signed CLA, let's close this PR. (It can be reopened).

**Comment by steveperry-53** [comment]
Closing because we've waited 27 days for the license to be signed. Reopen if you want to sign the license and move forward with this.

**Pull request closed without merging**